ILRepack/ILRepack.cs

```diff
@@ -352,7 +352,9 @@ public void Repack()
                 Logger.Info("Output directory does not exist. Creating output directory: " + outputDir);
                 Directory.CreateDirectory(outputDir);
             }
-            TargetAssemblyDefinition.Write(Options.OutputFile, parameters);
+            using (FileStream outputStream = File.OpenWrite(Options.OutputFile))
+                TargetAssemblyDefinition.Write(outputStream, parameters);
```

Comment:
Hmm, no, it calls the AssemblyDefinition one AFAICT. There doesn't seem to be any relation between two (except that the assembly contains modules)